test/integration/kubernetes_deploy_test.rb

```diff
@@ -52,6 +52,22 @@ def test_service_account_predeployed_before_unmanaged_pod
     ], in_order: true)
   end
 
+  def test_role_binding_predeployed_before_unmanaged_pod
+    result = deploy_fixtures("hello-cloud",
+      subset: ["configmap-data.yml", "unmanaged-pod.yml.erb", "role-binding.yml", "service-account.yml"])
+
+    # Expect that role binding account is deployed before the unmanaged pod
+    assert_deploy_success(result)
+    hello_cloud = FixtureSetAssertions::HelloCloud.new(@namespace)
+    hello_cloud.assert_configmap_data_present
+    hello_cloud.assert_all_service_accounts_up
```

Comment:
Can you please create the equivalent of this method for role bindings in `test/helpers/fixture_sets/hello_cloud.rb`, call it after this line, and also call it in `HelloCloud#assert_all_up` (that's used by `test_full_hello_cloud_set_deploy_succeeds`)?